### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gurpreetsingh00885**
Direction for contributors to test changes before committing them
is added. Also all the steps make sure python3 is being used for
everything (including the virtual environment).

Added build status badge too.

### CODE REVIEW

**Comment by gurpreetsingh00885** [comment]
Corrected @divyam3897

File: README.md [hunk 1]
```diff
@@ -63,8 +63,13 @@ This is the official website of Programming club, UIET, Panjab University
     ```
 
 11. Make your changes.
```
Comment (divyam3897) [inline README.md]: Shouldn't it be on one line? "Make your changes, execute the tests...."?

Comment (divyam3897) [reply README.md]: Did you fix these? I still dont see the changes :/

**Comment by gurpreetsingh00885** [comment]
@divyam3897 done making changes.

**Pull request merged**